docs/FAQ

```diff
@@ -1256,10 +1256,11 @@ FAQ
   libcurl will reuse connections for all transfers that are made using the
   same libcurl handle.
 
-  When you use the easy interface, the connection cache is kept within the
-  easy handle. If you instead use the multi interface, the connection cache
-  will be kept within the multi handle and will be shared among all the easy
-  handles that are used within the same multi handle.
+  When you use the easy interface curl_easy_setopt(), curl_easy_perform() the 
+  connection cache  is kept within the easy handle. If you instead use the 
```

Comment:
This doesn't make a proper sentence. Why introduce the function names in this paragraph?